addons/website_slides/models/slide_channel.py

```diff
@@ -570,40 +649,81 @@ def action_channel_invite(self):
             'res_model': 'slide.channel.invite',
             'target': 'new',
             'context': local_context,
+            'name': _('Share a Course')
```

Comment:
That would not allow to change course in the wizard, the title being set once for all. I don't know if that is an issue? If we keep the channel_id parameter, then it would also require a browse to get the name.